### PR TITLE
MySQL Prometheus dashboard KPI and layout updates

### PR DESCRIPTION
**Pull request by algchoo**
### Description
This pull request is updates the MySQL Prometheus integration dashboard.

### Changes
* [updated the layout and kpis for the mysql prometheus dashboard](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/669/commits/9e4594a7443aab6676d3811a576d51b790da6e56)
* [updated the mysql prometheus dashboard screenshot](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/669/commits/7dec2f3d543919745930e7cccffe2e3bb7d14c13)
* [updated the readme to reflect new metrics and chart titles](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/669/commits/f4c8e3ba5bc97ba755fadd4d20ff5af518e8a181)
* [updated the description for mysql prometheus dashboard in metadata](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/669/commits/4ad00684669e98000e54b3748a3314ce01a0d9eb)

### Updated screenshot
mysql-prometheus

### Additional KPIs
* Questions
* Insert/Update/Delete Statements
* Select Statements
* Fsync Operations
* InnoDB Buffer Pool Read Requests vs Reads From Disk

### CODE REVIEW

File: dashboards/mysql/metadata.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ sample_dashboards:
     id: mysql-prometheus
     display_name: MySQL Prometheus Overview
     description: |-
-      This dashboard has charts displaying: Global Status Uptime (days), Available Connections, Open Files Available, Cache Hit Rate, MySQL Up, Global Status Connection Error Rate (5m), Slow Queries, Buffer Pool Cache Hit Ratio, Full Join, Full Range Join, Range Check, and Scan
+      This dashboard has charts displaying: Uptime (days), Available Connections, Open Files Available, Cache Hit Rate, MySQL Up, Connection Errors, Slow Queries, Buffer Pool Cache Hit Rate, Questions, Insert/Update/Delete Statements, Select Statements, Fsync Operations, and InnoDB Buffer Pool Read Requests vs Reads From Disk
```
Comment (yqlu) [inline dashboards/mysql/metadata.yaml:17]: Just double-checking if this should match the README description exactly?

For example, the README correctly reflects that Uptime is now in minutes, not days.

File: integrations/mysql/prometheus_metadata.yaml [hunk 2]
```diff
@@ -73,4 +73,24 @@ platforms:
...
+        kind: CUMULATIVE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/mysql_global_status_innodb_data_fsyncs/unknown:counter
+        prometheus_name: mysql_global_status_innodb_data_fsyncs
+        kind: CUMULATIVE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/mysql_global_status_innodb_buffer_pool_reads/unknown:counter
+        prometheus_name: mysql_global_status_innodb_buffer_pool_reads
+        kind: CUMULATIVE
+        value_type: DOUBLE
```
Comment (yqlu) [inline integrations/mysql/prometheus_metadata.yaml:91]: This is already on line 56-59

File: dashboards/mysql/mysql-prometheus.json [hunk 3]
```diff
@@ -159,184 +169,292 @@
...
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(mysql_global_status_select_scan{${Cluster},${Location},${Namespace}}[5m])"
+                  "prometheusQuery": "rate(mysql_global_status_commands_total{command=~\"select\"}[5m])",
```
Comment (yqlu) [inline dashboards/mysql/mysql-prometheus.json:401]: Can you do a pass to ensure all the charts are consistently using the template variables? Seems like:
- Select statements
- Install | Update | Delete Statements
- Questions
- Fsync Operations
- InnoDB Buffer ...
aren't, so when I filter down to a single cluster or namespace, these still show the metrics for the entire project.

File: dashboards/mysql/mysql-prometheus.json [hunk 4]
```diff
@@ -20,34 +19,36 @@
...
+    "columns": 48,
     "tiles": [
       {
-        "height": 3,
+        "height": 8,
         "widget": {
           "scorecard": {
             "gaugeView": {
+              "lowerBound": 0,
               "upperBound": 1440
```
Comment (yqlu) [inline dashboards/mysql/mysql-prometheus.json:30]: Just double-checking whether it is correct and intentional for this have a gauge with thresholds? If uptime > 1440min, the card shows up as red. Do we want this, or just a scorecard?
Screenshot 2024-01-24 at 11 22 46 AM

Comment (algchoo) [reply dashboards/mysql/mysql-prometheus.json]: I agree with making this just a scorecard, it being red beyond a certain point is confusing. I've also updated the cache hit ratio scorecards to more clearly represent that they are percentages.

File: dashboards/mysql/metadata.yaml (hunk 1 above)
Comment (algchoo) [reply dashboards/mysql/metadata.yaml]: Thanks for catching this, it definitely should be minutes here.

**Comment by yqlu** [comment]
OK, can you regenerate the screenshot to reflect the gauges -> scorecards change? Everything else LGTM, thanks!

**Pull request merged**